xbmc/guilib/GUIAudioManager.cpp

```diff
@@ -110,7 +116,7 @@ void CGUIAudioManager::PlayActionSound(const CAction& action)
   if (!m_bEnabled)
     return;
 
-  actionSoundMap::iterator it = m_actionSoundMap.find(action.GetID());
+  const auto& it = m_actionSoundMap.find(action.GetID());
```

Comment:
getting a lightweight iterator returned here we don't need a reference, just `const auto it` ?